lib/logstash/agent.rb

```diff
@@ -321,10 +321,20 @@ def local_config(path)
         encoding_issue_files << file
       end
       config << cfg + "\n"
+      if config_test?
+        @logger.debug? && @logger.debug("\n***** Begin configuration file: #{file}")
```

Comment:
Why the newlines?

Also, any thoughts on making this look more like a valid config, so in this case using a comment `#` symbol to prefix the '***** Begin con....' lines?